Regression/WarpX-tests.ini

```
@@ -1245,6 +1245,7 @@ analysisRoutine = Examples/analysis_default_regression.py
 buildDir = .
 inputFile = Examples/Physics_applications/laser_acceleration/inputs_2d
 runtime_params = warpx.refine_plasma=1
+amr.ref_ratio_vect = 2 1
```

I think in the rest of this file we write all input parameters added to `runtime_params` on one single line:
```suggestion
runtime_params = warpx.refine_plasma=1 amr.ref_ratio_vect = 2 1
```